<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoChat</class>
 <widget class="QWidget" name="VideoChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>464</y>
     <width>835</width>
     <height>33</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="ButtonFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="microButton">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/microphoneOn.png</normaloff>:/imge/icon/png/microphoneOn.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="videoButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/videoOn.png</normaloff>:/imge/icon/png/videoOn.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="redioButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/redioOn.png</normaloff>:/imge/icon/png/redioOn.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="screenShareButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/screenSharing.png</normaloff>:/imge/icon/png/screenSharing.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="remoteButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/remoteControl.png</normaloff>:/imge/icon/png/remoteControl.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="manListButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/manmber.png</normaloff>:/imge/icon/png/manmber.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="chatButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/chatIcon.png</normaloff>:/imge/icon/png/chatIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="settingButton">
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/Setting.png</normaloff>:/imge/icon/png/Setting.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="overButton">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"> background-color:transparent;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/imge/icon/png/over.png</normaloff>:/imge/icon/png/over.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="previouPageBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imge/myIcon/previousPage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="nextPageBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imge/myIcon/nextPage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
